tasks/options/webpack.js

```diff
@@ -7,7 +7,7 @@ module.exports = function(grunt) {
     config.devtool = grunt.option('webpack-devtool') || 'cheap-source-map';
 
     config.module.rules = config.module.rules.map((rule) => {
-        if (rule.loader === 'ts-loader') {
+        if (rule.loader === 'ts-loader' && rule.test.test('index.ts')) {
```

Comment:
You don't need this check, let Webpack set `rule.options.transpileOnly = true` for all `ts-loader`s

Reply:
right